From 2a11d84e837ae0dc636748fe26d5251342fd320c Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 4 Nov 2016 21:35:04 +0300
Subject: [PATCH 18/24] ARM64: dts: r8a7796-salvator-x-view: Use
 r8a77xx-panel.dtsi

Use r8a77xx-panel.dtsi for LVDS LCD panel configuration.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 .../boot/dts/renesas/r8a7796-salvator-x-view.dts   | 35 ++++++----------------
 1 file changed, 9 insertions(+), 26 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796-salvator-x-view.dts b/arch/arm64/boot/dts/renesas/r8a7796-salvator-x-view.dts
index 4c3575e..3cb4207 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796-salvator-x-view.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7796-salvator-x-view.dts
@@ -33,6 +33,7 @@
 
 /dts-v1/;
 #include "r8a7796.dtsi"
+#include "r8a77xx-panel.dtsi"
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/input/input.h>
 
@@ -260,32 +261,6 @@
 		};
 	};
 
-	lvds {
-		compatible = "lvds-connector";
-
-		width-mm = <210>;
-		height-mm = <158>;
-
-		panel-timing {
-			/* 1024x768 @60Hz */
-			clock-frequency = <65000000>;
-			hactive = <1024>;
-			vactive = <768>;
-			hsync-len = <136>;
-			hfront-porch = <20>;
-			hback-porch = <160>;
-			vfront-porch = <3>;
-			vback-porch = <29>;
-			vsync-len = <6>;
-		};
-
-		port {
-			lvds_in: endpoint {
-				remote-endpoint = <&lvds_enc_out>;
-			};
-		};
-	};
-
 	hdmi-encoder {
 		compatible = "rockchip,rcar-dw-hdmi";
 
@@ -380,6 +355,14 @@
 	};
 };
 
+&lvds {
+	port {
+		lvds_in: endpoint {
+			remote-endpoint = <&lvds_enc_out>;
+		};
+	};
+};
+
 &pwm1 {
 	pinctrl-0 = <&pwm1_pins>;
 	pinctrl-names = "default";
-- 
1.9.3

